java/kotlin-extractor2/src/main/kotlin/entities/Expression.kt

```diff
@@ -215,25 +221,57 @@ OLD: KE1
         }
 */
 
-private fun isFunction(
+private fun isFunctionMatchingNames(
+    symbol: KaFunctionSymbol,
+    name: CallableId,
+    isExtension: Boolean = false,
+    nullability: KaTypeNullability = KaTypeNullability.UNKNOWN,
+    extensionReceiverClassName: ClassId? = null
+): Boolean {
```

Comment:
I was wondering if this would read more clearly as an extension, so you'd have `symbol.isFunctionMatchingNames(name, isExtension, nullability, extensionReceiverClassName)`

Reply:
I pushed an extra commit that changed the functions to extensions.